### PR TITLE
Laetitia Rework

### PR DESCRIPTION
**Pull request by PotatoTomahto**
## About The Pull Request

Changes how Laetitia's prank works.
- 70% to gain the gift on Neutral is unchanged
- 30% to remove gift on Good work, 15% to remove gift and 15% to begin explosion on Neutral, 70% to begin explosion on Bad
- If you work another abnormality with the gift, the explosion timer is halved
- Randomed gift timer between 3 to 4 minutes
- If you have Prudence 60, you get at most a 10 second warning before explosion.
- If you have Prudence 80, you are notified when you get the gift.

Added gift message and a neat indicator as a warning

Adds info to records, fixes consistency of some HE records.
## Why It's Good For The Game
Her prank used to be a 200 damage that was given out at a high chance, forcing people to stop their entire grind and wait it out with red armor. Because this was way too unpredictable and punishing, she was rarely chosen and worked significantly less than other HEs. This PR is designed to give people a chance to deal with and react to the prank, while still keeping the RNG surprise element. 
Laetitia is now an abnorm that punishes you if you're given her gift and work other abnorms, while also encouraging players to continue working her. Basically, be ready to keep working with her if you get a neutral ~~because she's a good girl who does nothing wrong~~.

## Changelog
:cl:
tweak: laetitia
:cl:

### CODE REVIEW

**Comment by Kitsunemitsu** [comment]
Remove the warning time, it should be a jumpscare.

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
See above comment.

**Comment by Kitsunemitsu** [comment]
Waiting on the rework from REDACTED, honestly. IDK Why you didn't wait until his PR was merged like I recommended.

File: code/modules/mob/living/simple_animal/abnormality/he/laetitia.dm [hunk 1]
```diff
@@ -22,15 +22,50 @@
...
+	var/datum/status_effect/pranked/P = user.has_status_effect(STATUS_EFFECT_PRANKED)
+	if(P)
+		if(prob(15)) //15% chance to remove prank
+			user.remove_status_effect(STATUS_EFFECT_PRANKED)
+		else if(prob(15)) //15% chance to trigger explosion
+			P.triggerprank()
+	else
+		if(prob(70)) //Do not give prank if tricked, not 100% of the time to be funny
+			user.apply_status_effect(STATUS_EFFECT_PRANKED)
+			RegisterSignal(user, COMSIG_WORK_STARTED, .proc/WorkCheck)
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/he/laetitia.dm:37]: That signal should probably go to the status effect itself.

File: code/modules/mob/living/simple_animal/abnormality/he/laetitia.dm [hunk 2]
```diff
@@ -59,25 +81,73 @@
...
-		L.apply_damage(200, RED_DAMAGE, null, L.run_armor_check(null, RED_DAMAGE), spread_damage = TRUE)//Usually a kill, you can block it if you're good
 
-#undef STATUS_EFFECT_TRICKED
+/datum/status_effect/pranked/on_apply()
+	if(get_attribute_level(owner, PRUDENCE_ATTRIBUTE) >= 80)
+		to_chat(owner, "<span class='warning'>You feel something slipped into your pocket.</span>")
+	RegisterSignal(owner, COMSIG_WORK_STARTED, .proc/WorkCheck)
+	return ..()
+
+/datum/status_effect/pranked/tick()
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/he/laetitia.dm:103]: Shouldn't there be parent call?

File: code/modules/mob/living/simple_animal/abnormality/he/laetitia.dm [hunk 3]
```diff
@@ -59,25 +81,73 @@
...
+				prank_overlay.layer = -BODY_FRONT_LAYER
+				prank_overlay.plane = FLOAT_PLANE
+				prank_overlay.mouse_opacity = 0
+				prank_overlay.vis_flags = VIS_INHERIT_ID
+				prank_overlay.alpha = 0
+				to_chat(L, "<span class='danger'>Your heart-shaped present begins to crack...</span>")
+				animate(prank_overlay, alpha = 255, time = (duration - world.time))
+				L.vis_contents += prank_overlay
+
+/datum/status_effect/pranked/on_remove()
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/he/laetitia.dm:121]: Same here. Missing parent call, I assume.

File: code/modules/paperwork/records/info/waw.dm [hunk 4]
```diff
@@ -321,27 +321,3 @@
 	<h4>Black Resistance:</h4> Weak	<br>
 	<h4>Pale Resistance:</h4> Fatal	<br>"}
 
-//Alriune
-/obj/item/paper/fluff/info/waw/alriune
```
Comment (EgorDinamit) [inline code/modules/paperwork/records/info/waw.dm:325]: You accidentally deleted records...

File: code/modules/mob/living/simple_animal/abnormality/he/laetitia.dm [hunk 5]
```diff
@@ -59,25 +81,73 @@
...
+	UnregisterSignal(owner, COMSIG_WORK_STARTED)
+	if(prank_overlay in owner.vis_contents)
+		owner.vis_contents -= prank_overlay
+	if(duration < world.time) //if prank removed due to it expiring
```
Comment (dsmith328) [inline code/modules/mob/living/simple_animal/abnormality/he/laetitia.dm:125]: `<=` might be better here

File: code/modules/paperwork/records/info/he.dm [hunk 6]
```diff
@@ -246,7 +246,7 @@
 	Work Damage : High	<br>
 	- When the work result was Bad, the Qliphoth Counter lowered.    <br>
 	- When an employee with Temperance Level 3 or higher completed their work, the Qliphoth Counter lowered.    <br>
-	- When an employee enters attempts work while the Counter is at 1, they will immediately die. Afterwards, the Qliphoth counter decreased and the Woodsman began searching for hearts.    <br>
+	- When an employee enters attempts work while the Counter is at 1, they would immediately die. Afterwards, the Qliphoth counter decreased and the Woodsman began searching for hearts.    <br>
```
Comment (dsmith328) [inline code/modules/paperwork/records/info/he.dm:249]: please remove the word `enters`

File: code/modules/paperwork/records/info/waw.dm (hunk 4 above)
Comment (dsmith328) [reply code/modules/paperwork/records/info/waw.dm:325]: See now it's gone... forever gone!

**Review by EgorDinamit — DISMISSED** [review]

**Review by EgorDinamit — APPROVED** [review]

**Review by Kitsunemitsu — APPROVED** [review]
Yeah if it's 10 seconds it's good.
I approve this from a balance perspective

**Pull request merged**